xwiki-platform-core/xwiki-platform-panels/xwiki-platform-panels-ui/src/main/resources/Panels/Applications.xml

```diff
@@ -551,18 +551,14 @@ require(['scriptaculous/effects'], function() {
 }
 
 .applicationPanelMoreButton {
-  padding: 10px 15px 10px 0;
+  padding: 10px 0 0 0;
   border: 0;
```

Comment:
Related to https://github.com/xwiki/xwiki-platform/pull/2262/files#r1272322198.
This does not matter much and can be ignored or dismissed without a discussion.
```suggestion
  border: 0;
  border-radius: 0.2em;
```

Reply:
Irrelevant as of now.